<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    
    <title>Full-length sequencing of circular DNA viruses and extrachromosomal circular DNA using CIDER-Seq | Luc Cornet</title>
    <meta name="viewport" content="width=device-width,minimum-scale=1">
    <meta name="description" content="Devang Mehta, Luc Cornet, Matthias Hirsch-Hoffmann, Syed Shan-e-Ali Zaidi and Hervé Vanderschuren
Circular DNA is ubiquitous in nature in the form of plasmids, circular DNA viruses, and extrachromosomal circular DNA (eccDNA) in eukaryotes. Sequencing of such molecules is essential to profiling virus distributions, discovering new viruses and understanding the roles of eccDNAs in eukaryotic cells. Circular DNA enrichment sequencing (CIDER-Seq) is a technique to enrich and accurately sequence circular DNA without the need for polymerase chain reaction amplification, cloning, and computational sequence assembly. The approach is based on randomly primed circular DNA amplification, which is followed by several enzymatic DNA repair steps and then by long-read sequencing. CIDER-Seq includes a custom data analysis package (CIDER-Seq Data Analysis Software 2) that implements the DeConcat algorithm to deconcatenate the long sequencing products of random circular DNA amplification into the intact sequences of the input circular DNA. The CIDER-Seq data analysis package can generate full-length annotated virus genomes, as well as circular DNA sequences of novel viruses. Applications of CIDER-Seq also include profiling of eccDNA molecules such as transposable elements (TEs) from biological samples. The method takes ~2 weeks to complete, depending on the computational resources available. Owing to the present constraints of long-read single-molecule sequencing, the accuracy of circular virus and eccDNA sequences generated by the CIDER-Seq method scales with sequence length, and the greatest accuracy is obtained for molecules &lt;10 kb long.
https://www.nature.com/articles/s41596-020-0301-0">
    <meta name="generator" content="Hugo 0.152.2">
    
    
    
      <meta name="robots" content="index, follow">
    
    

    
<link rel="stylesheet" href="/ananke/css/main.min.efe4d852f731d5d1fbb87718387202a97aafd768cdcdaed0662bbe6982e91824.css" >




    


    
      

    

    

    
      <link rel="canonical" href="https://lcornet.github.io/publications/cider/">
    

    
    
    <meta property="og:url" content="https://lcornet.github.io/publications/cider/">
  <meta property="og:site_name" content="Luc Cornet">
  <meta property="og:title" content="Full-length sequencing of circular DNA viruses and extrachromosomal circular DNA using CIDER-Seq">
  <meta property="og:description" content="Devang Mehta, Luc Cornet, Matthias Hirsch-Hoffmann, Syed Shan-e-Ali Zaidi and Hervé Vanderschuren
Circular DNA is ubiquitous in nature in the form of plasmids, circular DNA viruses, and extrachromosomal circular DNA (eccDNA) in eukaryotes. Sequencing of such molecules is essential to profiling virus distributions, discovering new viruses and understanding the roles of eccDNAs in eukaryotic cells. Circular DNA enrichment sequencing (CIDER-Seq) is a technique to enrich and accurately sequence circular DNA without the need for polymerase chain reaction amplification, cloning, and computational sequence assembly. The approach is based on randomly primed circular DNA amplification, which is followed by several enzymatic DNA repair steps and then by long-read sequencing. CIDER-Seq includes a custom data analysis package (CIDER-Seq Data Analysis Software 2) that implements the DeConcat algorithm to deconcatenate the long sequencing products of random circular DNA amplification into the intact sequences of the input circular DNA. The CIDER-Seq data analysis package can generate full-length annotated virus genomes, as well as circular DNA sequences of novel viruses. Applications of CIDER-Seq also include profiling of eccDNA molecules such as transposable elements (TEs) from biological samples. The method takes ~2 weeks to complete, depending on the computational resources available. Owing to the present constraints of long-read single-molecule sequencing, the accuracy of circular virus and eccDNA sequences generated by the CIDER-Seq method scales with sequence length, and the greatest accuracy is obtained for molecules &lt;10 kb long.
https://www.nature.com/articles/s41596-020-0301-0">
  <meta property="og:locale" content="en_us">
  <meta property="og:type" content="article">
    <meta property="article:section" content="publications">
    <meta property="article:tag" content="2020">
    <meta property="article:tag" content="Plant Genomics">

  <meta itemprop="name" content="Full-length sequencing of circular DNA viruses and extrachromosomal circular DNA using CIDER-Seq">
  <meta itemprop="description" content="Devang Mehta, Luc Cornet, Matthias Hirsch-Hoffmann, Syed Shan-e-Ali Zaidi and Hervé Vanderschuren
Circular DNA is ubiquitous in nature in the form of plasmids, circular DNA viruses, and extrachromosomal circular DNA (eccDNA) in eukaryotes. Sequencing of such molecules is essential to profiling virus distributions, discovering new viruses and understanding the roles of eccDNAs in eukaryotic cells. Circular DNA enrichment sequencing (CIDER-Seq) is a technique to enrich and accurately sequence circular DNA without the need for polymerase chain reaction amplification, cloning, and computational sequence assembly. The approach is based on randomly primed circular DNA amplification, which is followed by several enzymatic DNA repair steps and then by long-read sequencing. CIDER-Seq includes a custom data analysis package (CIDER-Seq Data Analysis Software 2) that implements the DeConcat algorithm to deconcatenate the long sequencing products of random circular DNA amplification into the intact sequences of the input circular DNA. The CIDER-Seq data analysis package can generate full-length annotated virus genomes, as well as circular DNA sequences of novel viruses. Applications of CIDER-Seq also include profiling of eccDNA molecules such as transposable elements (TEs) from biological samples. The method takes ~2 weeks to complete, depending on the computational resources available. Owing to the present constraints of long-read single-molecule sequencing, the accuracy of circular virus and eccDNA sequences generated by the CIDER-Seq method scales with sequence length, and the greatest accuracy is obtained for molecules &lt;10 kb long.
https://www.nature.com/articles/s41596-020-0301-0">
  <meta itemprop="wordCount" content="232">
  <meta itemprop="keywords" content="2020,Plant Genomics">
  <meta name="twitter:card" content="summary">
  <meta name="twitter:title" content="Full-length sequencing of circular DNA viruses and extrachromosomal circular DNA using CIDER-Seq">
  <meta name="twitter:description" content="Devang Mehta, Luc Cornet, Matthias Hirsch-Hoffmann, Syed Shan-e-Ali Zaidi and Hervé Vanderschuren
Circular DNA is ubiquitous in nature in the form of plasmids, circular DNA viruses, and extrachromosomal circular DNA (eccDNA) in eukaryotes. Sequencing of such molecules is essential to profiling virus distributions, discovering new viruses and understanding the roles of eccDNAs in eukaryotic cells. Circular DNA enrichment sequencing (CIDER-Seq) is a technique to enrich and accurately sequence circular DNA without the need for polymerase chain reaction amplification, cloning, and computational sequence assembly. The approach is based on randomly primed circular DNA amplification, which is followed by several enzymatic DNA repair steps and then by long-read sequencing. CIDER-Seq includes a custom data analysis package (CIDER-Seq Data Analysis Software 2) that implements the DeConcat algorithm to deconcatenate the long sequencing products of random circular DNA amplification into the intact sequences of the input circular DNA. The CIDER-Seq data analysis package can generate full-length annotated virus genomes, as well as circular DNA sequences of novel viruses. Applications of CIDER-Seq also include profiling of eccDNA molecules such as transposable elements (TEs) from biological samples. The method takes ~2 weeks to complete, depending on the computational resources available. Owing to the present constraints of long-read single-molecule sequencing, the accuracy of circular virus and eccDNA sequences generated by the CIDER-Seq method scales with sequence length, and the greatest accuracy is obtained for molecules &lt;10 kb long.
https://www.nature.com/articles/s41596-020-0301-0">

      
      
    
	
  </head><body class="ma0 avenir bg-near-white production">

    
   
  

  <header>
    <div class="bg-black">
      <nav class="pv3 ph3 ph4-ns" role="navigation">
  <div class="flex-l center items-center justify-between">
    <a href="/" class="f3 fw2 hover-white white-90 dib no-underline">
      
        Luc Cornet
      
    </a>
    <div class="flex-l items-center">
      

      
        <ul class="pl0 mr3">
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white white-90 no-underline" href="/cv/" title="CV page">
              CV
            </a>
          </li>
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white white-90 no-underline" href="/publications/" title="Publications page">
              Publications
            </a>
          </li>
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white white-90 no-underline" href="/projects/" title="Projects page">
              Projects
            </a>
          </li>
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white white-90 no-underline" href="/citizen/" title="Citizen-Science page">
              Citizen-Science
            </a>
          </li>
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white white-90 no-underline" href="/basal/" title="BASAL-team page">
              BASAL-team
            </a>
          </li>
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white white-90 no-underline" href="/join/" title="Join-us page">
              Join-us
            </a>
          </li>
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white white-90 no-underline" href="/contact/" title="Contact page">
              Contact
            </a>
          </li>
          
        </ul>
      
      <div class="ananke-socials"></div>

    </div>
  </div>
</nav>

    </div>
  </header>



    <main class="pb7" role="main">
      
  
  <article class="flex-l mw7 center ph3 flex-wrap justify-between">
    <header class="mt4 w-100">
      <aside class="instapaper_ignoref b helvetica tracked ttu">
          
        Publications
      </aside><div id="sharing" class="mt3 ananke-socials"></div>
<h1 class="f1 athelas mt3 mb1">Full-length sequencing of circular DNA viruses and extrachromosomal circular DNA using CIDER-Seq</h1>
      
      
      

      
      
    </header>
    <div class="nested-copy-line-height lh-copy serif f4 nested-links mid-gray pr4-l w-100-l"><p><strong>Devang Mehta, Luc Cornet, Matthias Hirsch-Hoffmann, Syed Shan-e-Ali Zaidi and Hervé Vanderschuren</strong><br>
Circular DNA is ubiquitous in nature in the form of plasmids, circular DNA viruses, and extrachromosomal circular DNA (eccDNA) in eukaryotes. Sequencing of such molecules is essential to profiling virus distributions, discovering new viruses and understanding the roles of eccDNAs in eukaryotic cells. Circular DNA enrichment sequencing (CIDER-Seq) is a technique to enrich and accurately sequence circular DNA without the need for polymerase chain reaction amplification, cloning, and computational sequence assembly. The approach is based on randomly primed circular DNA amplification, which is followed by several enzymatic DNA repair steps and then by long-read sequencing. CIDER-Seq includes a custom data analysis package (CIDER-Seq Data Analysis Software 2) that implements the DeConcat algorithm to deconcatenate the long sequencing products of random circular DNA amplification into the intact sequences of the input circular DNA. The CIDER-Seq data analysis package can generate full-length annotated virus genomes, as well as circular DNA sequences of novel viruses. Applications of CIDER-Seq also include profiling of eccDNA molecules such as transposable elements (TEs) from biological samples. The method takes ~2 weeks to complete, depending on the computational resources available. Owing to the present constraints of long-read single-molecule sequencing, the accuracy of circular virus and eccDNA sequences generated by the CIDER-Seq method scales with sequence length, and the greatest accuracy is obtained for molecules &lt;10 kb long.<br>
<a href="https://www.nature.com/articles/s41596-020-0301-0">https://www.nature.com/articles/s41596-020-0301-0</a></p>
<ul class="pa0">
  
   <li class="list di">
     <a href="/tags/2020/" class="link f5 grow br-pill ba ph3 pv2 mb2 dib black sans-serif no-underline">2020</a>
   </li>
  
   <li class="list di">
     <a href="/tags/plant-genomics/" class="link f5 grow br-pill ba ph3 pv2 mb2 dib black sans-serif no-underline">Plant Genomics</a>
   </li>
  
</ul>
<div class="mt6 instapaper_ignoref">
      
      
      </div>
    </div></article>

    </main>
    <footer class="bg-black bottom-0 w-100 pa3" role="contentinfo">
  <div class="flex justify-between">
  <a class="f4 fw4 hover-white white-70 dn dib-ns pv2 ph3 no-underline" href="https://lcornet.github.io/" >
    &copy;  Luc Cornet 2026 
  </a>
    <div><div class="ananke-socials"></div>
</div>
  </div>
</footer>

  </body>
</html>
