<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    
    <title>Luc Cornet</title>
    <meta name="viewport" content="width=device-width,minimum-scale=1">
    <meta name="description" content="">
    <meta name="generator" content="Hugo 0.152.2">
    
    
    
      <meta name="robots" content="index, follow">
    
    

    
<link rel="stylesheet" href="/ananke/css/main.min.efe4d852f731d5d1fbb87718387202a97aafd768cdcdaed0662bbe6982e91824.css" >




    


    
      

    

    
    
      <link href="/tags/cyanobacteria/index.xml" rel="alternate" type="application/rss+xml" title="Luc Cornet" />
      <link href="/tags/cyanobacteria/index.xml" rel="feed" type="application/rss+xml" title="Luc Cornet" />
      
    

    
      <link rel="canonical" href="https://lcornet.github.io/tags/cyanobacteria/">
    

    
    
    <meta property="og:url" content="https://lcornet.github.io/tags/cyanobacteria/">
  <meta property="og:site_name" content="Luc Cornet">
  <meta property="og:title" content="Cyanobacteria">
  <meta property="og:locale" content="en_us">
  <meta property="og:type" content="website">

  <meta itemprop="name" content="Cyanobacteria">
  <meta name="twitter:card" content="summary">
  <meta name="twitter:title" content="Cyanobacteria">

      
      
    
	
  </head><body class="ma0 avenir bg-near-white production">

    

  <header>
    <div class="pb3-m pb6-l bg-black">
      <nav class="pv3 ph3 ph4-ns" role="navigation">
  <div class="flex-l center items-center justify-between">
    <a href="/" class="f3 fw2 hover-white white-90 dib no-underline">
      
        Luc Cornet
      
    </a>
    <div class="flex-l items-center">
      

      
        <ul class="pl0 mr3">
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white white-90 no-underline" href="/cv/" title="CV page">
              CV
            </a>
          </li>
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white white-90 no-underline" href="/publications/" title="Publications page">
              Publications
            </a>
          </li>
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white white-90 no-underline" href="/projects/" title="Projects page">
              Projects
            </a>
          </li>
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white white-90 no-underline" href="/citizen/" title="Citizen Science page">
              Citizen Science
            </a>
          </li>
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white white-90 no-underline" href="/basal/" title="BASAL Team page">
              BASAL Team
            </a>
          </li>
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white white-90 no-underline" href="/join/" title="Join Us page">
              Join Us
            </a>
          </li>
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white white-90 no-underline" href="/contact/" title="Contact page">
              Contact
            </a>
          </li>
          
        </ul>
      
      <div class="ananke-socials"></div>

    </div>
  </div>
</nav>

      <div class="tc-l pv3 ph3 ph4-ns">
        <h1 class="f2 f-subheadline-l fw2 light-silver mb0 lh-title">
          Cyanobacteria
        </h1>
        
      </div>
    </div>
  </header>


    <main class="pb7" role="main">
      
  <article class="pa3 pa4-ns nested-copy-line-height">
    <section class="cf ph3 ph5-l pv3 pv4-l f4 tc-l center measure-wide lh-copy nested-links mid-gray"></section>
    <section class="flex-ns mt5 flex-wrap justify-around">
      
        <div class="w-100 w-30-l mb4 relative bg-white">
          <div class="w-100 mb4 nested-copy-line-height relative bg-white">
  <div class="mb3 pa4 gray overflow-hidden bg-white">
    <span class="f6 db">Publications</span>
    <h1 class="f3 near-black">
      <a href="/publications/theory/" class="link black dim">
        On the non-oxygenic origins of thylakoids
      </a>
    </h1>
    <div class="nested-links f5 lh-copy nested-copy-line-height">
      <p><strong>Luc Cornet</strong><br>
Thylakoid membranes are the site of oxygenic photosynthesis, one of the most important biochemical processes on earth. The ancestral state of these membranes is represented today in Gloeobacterales, where they are lacking and photosynthesis instead takes place in the cytoplasmic membrane. The evolutionary transition from this ancestral state to the modern thylakoid membranes provided a major advantage, as it increased photosynthetic efficiency. However, how this significant transition occurred remains an understudied question. The biogenesis of modern thylakoid membranes relies on a highly synchronized process involving numerous assembly factors and showing important differences between the two photosystems. Together, these features suggest the existence of intermediate evolutionary states during the emergence of this compartment. Here, I propose a non-oxygenic origin of thylakoid membranes, where these intermediate states were initially dedicated to alternative electron flows. This hypothesis further addresses the paradox of cyanobacterial diversification in an euxinic environment, toxic to photosystem II.<br>
<a href="https://www.nature.com/articles/s42003-025-09100-w">https://www.nature.com/articles/s42003-025-09100-w</a></p>
    </div>
    <a href="/publications/theory/" class="ba b--moon-gray bg-light-gray br2 color-inherit dib f7 hover-bg-moon-gray link mt2 ph2 pv1">read more</a>
  </div>
</div>

        </div>
      
        <div class="w-100 w-30-l mb4 relative bg-white">
          <div class="w-100 mb4 nested-copy-line-height relative bg-white">
  <div class="mb3 pa4 gray overflow-hidden bg-white">
    <span class="f6 db">Publications</span>
    <h1 class="f3 near-black">
      <a href="/publications/raspam/" class="link black dim">
        A FAIR Amplicon Sequencing Workflow for Long-term Environmental Monitoring
      </a>
    </h1>
    <div class="nested-links f5 lh-copy nested-copy-line-height">
      <p><strong>Marie Harmel, Benoit Durieu, Valentina Savaglia, Bjorn Tytgat, Denis Baurain, Annick Wilmotte, Elie Verleyen, Luc Cornet</strong><br>
Antarctica represents one of the last pristine environments on Earth, providing a unique opportunity to to study the effects of climate change and anthropogenic activities. Ice-free areas, such as the inland nunataks of the Sør Rondane Mountains (SRM), host unique terrestrial and lacustrine ecosystems, of which the simplified food webs rely almost exclusively on microbial primary production. Because of their small size, low productivity and hence low biomass, these microbial communities are fragile. Seven SRM sites were selected to be part of the Antarctic Specially Protected Area (ASPA) 179. The MonASPA project has established an environmental monitoring program to evaluate the effectiveness of the management plan approved by the Antarctic Treaty System for these areas. A key component of MonASPA is the long-term monitoring of microbial biodiversity using 16S rRNA amplicon sequencing. To ensure consistent operation over decades, we developed the Reproducible Amplicon Sequencing Pipeline for Antarctic Monitoring (RASPAM), which is built on Apptainer containers and Nextflow workflows. RASPAM implements Amplicon Sequence Variants (ASVs) and Zero-radius Operational Taxonomic Units (ZOTUs) to provide high-resolution taxonomic affiliation. It incorporates, in addition to the SILVA database, a taxonomically curated 16S rRNA database for cyanobacteria and enables comparisons against NCBI databases to facilitate the identification of rare prokaryotic strains in environmental samples. RASPAM is Findable, Accessible, Interoperable, and Reproducible (FAIR) and represents a robust tool for long-term monitoring of microbial communities in Antarctic and other extreme environments.<br>
<a href="https://doi.org/10.64898/2025.12.04.692289">https://doi.org/10.64898/2025.12.04.692289</a></p>
    </div>
    <a href="/publications/raspam/" class="ba b--moon-gray bg-light-gray br2 color-inherit dib f7 hover-bg-moon-gray link mt2 ph2 pv1">read more</a>
  </div>
</div>

        </div>
      
        <div class="w-100 w-30-l mb4 relative bg-white">
          <div class="w-100 mb4 nested-copy-line-height relative bg-white">
  <div class="mb3 pa4 gray overflow-hidden bg-white">
    <span class="f6 db">Publications</span>
    <h1 class="f3 near-black">
      <a href="/publications/sqr/" class="link black dim">
        Early-Diverging SQR Enzyme in Antarctic Gloeobacterales Indicates Sulfide Tolerance in Thylakoid-Lacking Cyanobacteria
      </a>
    </h1>
    <div class="nested-links f5 lh-copy nested-copy-line-height">
      <p><strong>Louise Hambücken, Edi Sudianto, Jimmy H. Saw, Denis Baurain, Luc Cornet</strong> <br>
Oxygenic photosynthesis, which converts solar energy into carbohydrates via a linear electron transport chain and two photosystems (PSII and PSI), first appeared in cyanobacteria approximately 3.3 Ga and drove the Great Oxidation Event around 2.4 Ga. During this period, euxinic conditions—characterized by sulfidic, anoxic oceans—posed a metabolic challenge to cyanobacteria, as sulfide inhibits PSII, the reaction center responsible for water splitting. Here, we report the presence of an early-diverging form of the sulfide quinone reductase (SQR) enzyme in Antarctic representatives of Gloeobacterales, the earliest-branching cyanobacterial lineage lacking thylakoids. Phylogenetic analyses reveal that these SQR sequences are the earliest-diverging cyanobacterial SQR known to date, predating the multiple lateral gene transfer events previously observed in the phylum. Additional searches in metagenomic datasets indicate that such sequences are restricted to cold environments. Our findings unveil possible adaptive strategies of early cyanobacteria to cope with sulfidic stress and point to Antarctic lakes as preserved natural laboratories for investigating cyanobacterial diversification and the evolution of oxygenic photosynthesis under euxinic conditions.<br>
<a href="https://doi.org/10.1101/2025.10.24.684318">https://doi.org/10.1101/2025.10.24.684318</a></p>
    </div>
    <a href="/publications/sqr/" class="ba b--moon-gray bg-light-gray br2 color-inherit dib f7 hover-bg-moon-gray link mt2 ph2 pv1">read more</a>
  </div>
</div>

        </div>
      
        <div class="w-100 w-30-l mb4 relative bg-white">
          <div class="w-100 mb4 nested-copy-line-height relative bg-white">
  <div class="mb3 pa4 gray overflow-hidden bg-white">
    <span class="f6 db">Publications</span>
    <h1 class="f3 near-black">
      <a href="/publications/ribose/" class="link black dim">
        Horizontal Gene Transfers Underpin Ribose Heterotrophy and Central Carbon Metabolism Remodeling in Gloeobacteraceae
      </a>
    </h1>
    <div class="nested-links f5 lh-copy nested-copy-line-height">
      <p><strong>Edi Sudianto, Denis Baurain, Luc Cornet</strong> <br>
Gloeobacterales has long been considered a “living fossil” cyanobacterial order, owing to its lack of thylakoid membranes and basal phylogenetic position. However, our study reveals that Gloeobacterales actively integrate horizontally transferred genes into their core metabolism. In Gloeobacteraceae—one of the two families within the order—these genes encode a complete ribose ATP synthase binding cassette (ABC) importer and downstream enzymes, enabling the heterotrophic uptake of external ribose and its assimilation into central carbon metabolism, along with photosynthesis, indicative of photomixotrophy. Beyond ribose utilization, their central carbon metabolism exhibits a mosaic architecture shaped by the integration of foreign genes into the Calvin-Benson-Bassham cycle, the pentose phosphate pathway, and the Embden-Meyerhof-Parnas pathway. Uniquely, these genes appear to have been acquired through multiple independent transfer events, as reflected by their dispersed genomic locations and diverse bacterial donors, including other cyanobacteria and Pseudomonadota. These findings contradict the long-standing view of Gloeobacterales as metabolically primitive relics. Instead, Gloeobacterales is likely a dynamic lineage that continues to adapt and evolve through metabolic innovation and the assimilation of foreign genes into its genomes.<br>
<a href="https://doi.org/10.1101/2025.11.10.686926">https://doi.org/10.1101/2025.11.10.686926</a></p>
    </div>
    <a href="/publications/ribose/" class="ba b--moon-gray bg-light-gray br2 color-inherit dib f7 hover-bg-moon-gray link mt2 ph2 pv1">read more</a>
  </div>
</div>

        </div>
      
        <div class="w-100 w-30-l mb4 relative bg-white">
          <div class="w-100 mb4 nested-copy-line-height relative bg-white">
  <div class="mb3 pa4 gray overflow-hidden bg-white">
    <span class="f6 db">Publications</span>
    <h1 class="f3 near-black">
      <a href="/publications/pico/" class="link black dim">
        Out of Antarctica: new insights into Antarctic Subcluster 5.2 picocyanobacteria based on high-quality genomes
      </a>
    </h1>
    <div class="nested-links f5 lh-copy nested-copy-line-height">
      <p><strong>Benoit Durieu, Valentina Savaglia, Mick Van Vlierberghe, Valérian Lupo, Denis Baurain, Annick Wilmotte, Luc Cornet</strong><br>
Synechococcus-like cyanobacteria are cosmopolitan unicellular picocyanobacteria that have colonized diverse aquatic and terrestrial habitats. The so-called ‘subcluster 5.2’ represents a particularly diversified subgroup, including marine and freshwater organisms adapted to extreme conditions, notably polar environments. We increased the genomic representation of polar taxa in this subcluster by reconstructing new high-quality genomes from five Antarctic lacustrine strains and one Arctic freshwater isolate using a combination of small Illumina and long Nanopore reads. A maximum likelihood (ML) phylogenomic analysis of these new assemblies combined with all publicly available good quality assemblies of the subcluster 5.2 suggests evidence of a dispersal process from Antarctica. Indeed, the topology of the phylogenomic tree indicates one basal Antarctic lineage followed by the emergence of two lineages, one Antarctic and one non-Antarctic (Spain). This finding is further supported by a 16S rRNA ML phylogenetic and a pangenomic analysis. Although secondary colonization of Antarctica by cyanobacteria following the cooling of the continent 34 million years ago has been reported, this study is the first to support an ‘Out-of-Antarctica’ scenario inside subcluster 5.2.<br>
<a href="https://doi.org/10.64898/2025.12.17.694815">https://doi.org/10.64898/2025.12.17.694815</a></p>
    </div>
    <a href="/publications/pico/" class="ba b--moon-gray bg-light-gray br2 color-inherit dib f7 hover-bg-moon-gray link mt2 ph2 pv1">read more</a>
  </div>
</div>

        </div>
      
        <div class="w-100 w-30-l mb4 relative bg-white">
          <div class="w-100 mb4 nested-copy-line-height relative bg-white">
  <div class="mb3 pa4 gray overflow-hidden bg-white">
    <span class="f6 db">Publications</span>
    <h1 class="f3 near-black">
      <a href="/publications/orper/" class="link black dim">
        ORPER: A Workflow for Constrained SSU rRNA Phylogenies
      </a>
    </h1>
    <div class="nested-links f5 lh-copy nested-copy-line-height">
      <p><strong>Luc Cornet, Anne-Catherine Ahn, Annick Wilmotte, and Denis Baurain</strong><br>
he continuous increase in sequenced genomes in public repositories makes the choice of interesting bacterial strains for future sequencing projects ever more complicated, as it is difficult to estimate the redundancy between these strains and the already available genomes. Therefore, we developed the Nextflow workflow “ORPER”, for “ORganism PlacER”, containerized in Singularity, which allows the determination the phylogenetic position of a collection of organisms in the genomic landscape. ORPER constrains the phylogenetic placement of SSU (16S) rRNA sequences in a multilocus reference tree based on ribosomal protein genes extracted from public genomes. We demonstrate the utility of ORPER on the Cyanobacteria phylum, by placing 152 strains of the BCCM/ULC collection.</p>
    </div>
    <a href="/publications/orper/" class="ba b--moon-gray bg-light-gray br2 color-inherit dib f7 hover-bg-moon-gray link mt2 ph2 pv1">read more</a>
  </div>
</div>

        </div>
      
        <div class="w-100 w-30-l mb4 relative bg-white">
          <div class="w-100 mb4 nested-copy-line-height relative bg-white">
  <div class="mb3 pa4 gray overflow-hidden bg-white">
    <span class="f6 db">Publications</span>
    <h1 class="f3 near-black">
      <a href="/publications/fossil/" class="link black dim">
        Cyanobacteria evolution: Insight from the fossil record
      </a>
    </h1>
    <div class="nested-links f5 lh-copy nested-copy-line-height">
      <p><strong>Catherine F. Demoulin, Yannick J. Lara, Luc Cornet, Camille François, Denis Baurain, Annick Wilmotte, Emmanuelle J. Javaux</strong><br>
Cyanobacteria played an important role in the evolution of Early Earth and the biosphere. They are responsible for the oxygenation of the atmosphere and oceans since the Great Oxidation Event around 2.4 Ga, debatably earlier. They are also major primary producers in past and present oceans, and the ancestors of the chloroplast. Nevertheless, the identification of cyanobacteria in the early fossil record remains ambiguous because the morphological criteria commonly used are not always reliable for microfossil interpretation. Recently, new biosignatures specific to cyanobacteria were proposed. Here, we review the classic and new cyanobacterial biosignatures. We also assess the reliability of the previously described cyanobacteria fossil record and the challenges of molecular approaches on modern cyanobacteria. Finally, we suggest possible new calibration points for molecular clocks, and strategies to improve our understanding of the timing and pattern of the evolution of cyanobacteria and oxygenic photosynthesis.<br>
<a href="https://doi.org/10.1016/j.freeradbiomed.2019.05.007">https://doi.org/10.1016/j.freeradbiomed.2019.05.007</a></p>
    </div>
    <a href="/publications/fossil/" class="ba b--moon-gray bg-light-gray br2 color-inherit dib f7 hover-bg-moon-gray link mt2 ph2 pv1">read more</a>
  </div>
</div>

        </div>
      
        <div class="w-100 w-30-l mb4 relative bg-white">
          <div class="w-100 mb4 nested-copy-line-height relative bg-white">
  <div class="mb3 pa4 gray overflow-hidden bg-white">
    <span class="f6 db">Publications</span>
    <h1 class="f3 near-black">
      <a href="/publications/ssu/" class="link black dim">
        A constrained SSU-rRNA phylogeny reveals the unsequenced diversity of photosynthetic Cyanobacteria (Oxyphotobacteria)
      </a>
    </h1>
    <div class="nested-links f5 lh-copy nested-copy-line-height">
      <p><strong>Luc Cornet, Annick Wilmotte, Emmanuelle J. Javaux and Denis Baurain</strong><br>
Cyanobacteria are an ancient phylum of prokaryotes that contain the class Oxyphotobacteria. This group has been extensively studied by phylogenomics notably because it is widely accepted that Cyanobacteria were responsible for the spread of photosynthesis to the eukaryotic domain. The aim of this study was to evaluate the fraction of the oxyphotobacterial diversity for which sequenced genomes are available for genomic studies. For this, we built a phylogenomic-constrained SSU rRNA (16S) tree to pinpoint unexploited clusters of Oxyphotobacteria that should be targeted for future genome sequencing, so as to improve our understanding of Oxyphotobacteria evolution.<br>
<a href="https://link.springer.com/article/10.1186/s13104-018-3543-y">https://link.springer.com/article/10.1186/s13104-018-3543-y</a></p>
    </div>
    <a href="/publications/ssu/" class="ba b--moon-gray bg-light-gray br2 color-inherit dib f7 hover-bg-moon-gray link mt2 ph2 pv1">read more</a>
  </div>
</div>

        </div>
      
        <div class="w-100 w-30-l mb4 relative bg-white">
          <div class="w-100 mb4 nested-copy-line-height relative bg-white">
  <div class="mb3 pa4 gray overflow-hidden bg-white">
    <span class="f6 db">Publications</span>
    <h1 class="f3 near-black">
      <a href="/publications/conta1/" class="link black dim">
        Consensus assessment of the contamination level of publicly available cyanobacterial genomes
      </a>
    </h1>
    <div class="nested-links f5 lh-copy nested-copy-line-height">
      <p><strong>Luc Cornet, Loïc Meunier, Mick Van Vlierberghe, Raphaël R. Léonard, Benoit Durieu, Yannick Lara, Agnieszka Misztak, Damien Sirjacobs, Emmanuelle J. Javaux, Hervé Philippe, Annick Wilmotte, Denis Baurain</strong><br>
Publicly available genomes are crucial for phylogenetic and metagenomic studies, in which contaminating sequences can be the cause of major problems. This issue is expected to be especially important for Cyanobacteria because axenic strains are notoriously difficult to obtain and keep in culture. Yet, despite their great scientific interest, no data are currently available concerning the quality of publicly available cyanobacterial genomes. As reliably detecting contaminants is a complex task, we designed a pipeline combining six methods in a consensus strategy to assess the contamination level of 440 genome assemblies of Cyanobacteria. Two methods are based on published reference databases of ribosomal genes (SSU rRNA 16S and ribosomal proteins), one is indirectly based on a reference database of marker genes (CheckM), and three are based on complete genome analysis. Among those genome-wide methods, Kraken and DIAMOND blastx share the same reference database that we derived from Ensembl Bacteria, whereas CONCOCT does not require any reference database, instead relying on differences in DNA tetramer frequencies. Given that all the six methods appear to have their own strengths and limitations, we used the consensus of their rankings to infer that &gt;5% of cyanobacterial genome assemblies are highly contaminated by foreign DNA (i.e., contaminants were detected by 5 or 6 methods). Our results will help researchers to check the quality of publicly available genomic data before use in their own analyses. Moreover, we argue that journals should make mandatory the submission of raw read data along with genome assemblies in order to facilitate the detection of contaminants in sequence databases.<br>
<a href="https://doi.org/10.1371/journal.pone.0200323">https://doi.org/10.1371/journal.pone.0200323</a></p>
    </div>
    <a href="/publications/conta1/" class="ba b--moon-gray bg-light-gray br2 color-inherit dib f7 hover-bg-moon-gray link mt2 ph2 pv1">read more</a>
  </div>
</div>

        </div>
      
        <div class="w-100 w-30-l mb4 relative bg-white">
          <div class="w-100 mb4 nested-copy-line-height relative bg-white">
  <div class="mb3 pa4 gray overflow-hidden bg-white">
    <span class="f6 db">Publications</span>
    <h1 class="f3 near-black">
      <a href="/publications/ulc007/" class="link black dim">
        Draft Genome Sequence of the Axenic Strain Phormidesmispriestleyi ULC007, a Cyanobacterium Isolated from Lake Bruehwiler (Larsemann Hills, Antarctica)
      </a>
    </h1>
    <div class="nested-links f5 lh-copy nested-copy-line-height">
      <p><strong>Yannick Lara, Benoit Durieu, Luc Cornet, Olivier Verlaine, Rosmarie Rippka, Igor S. Pessi, Agnieszka Misztak, Bernard Joris, Emmanuelle J. Javaux, Denis Baurain, Annick Wilmotte</strong><br>
Phormidesmis priestleyi ULC007 is an Antarctic freshwater cyanobacterium. Its draft genome is 5,684,389 bp long. It contains a total of 5,604 protein-encoding genes, of which 22.2% have no clear homologues in known genomes. To date, this draft genome is the first one ever determined for an axenic cyanobacterium from Antarctica.<br>
<a href="https://doi.org/10.1128/genomea.01546-16">https://doi.org/10.1128/genomea.01546-16</a></p>
    </div>
    <a href="/publications/ulc007/" class="ba b--moon-gray bg-light-gray br2 color-inherit dib f7 hover-bg-moon-gray link mt2 ph2 pv1">read more</a>
  </div>
</div>

        </div>
      
    </section>
    
    <ul class="pagination pagination-default">
      <li class="page-item disabled">
        <a aria-disabled="true" aria-label="First" class="page-link" role="button" tabindex="-1"><span aria-hidden="true">&laquo;&laquo;</span></a>
      </li>
      <li class="page-item disabled">
        <a aria-disabled="true" aria-label="Previous" class="page-link" role="button" tabindex="-1"><span aria-hidden="true">&laquo;</span></a>
      </li>
      <li class="page-item active">
        <a aria-current="page" aria-label="Page 1" class="page-link" role="button">1</a>
      </li>
      <li class="page-item">
        <a href="/tags/cyanobacteria/page/2/" aria-label="Page 2" class="page-link" role="button">2</a>
      </li>
      <li class="page-item">
        <a href="/tags/cyanobacteria/page/2/" aria-label="Next" class="page-link" role="button"><span aria-hidden="true">&raquo;</span></a>
      </li>
      <li class="page-item">
        <a href="/tags/cyanobacteria/page/2/" aria-label="Last" class="page-link" role="button"><span aria-hidden="true">&raquo;&raquo;</span></a>
      </li>
    </ul></article>

    </main>
    <footer class="bg-black bottom-0 w-100 pa3" role="contentinfo">
  <div class="flex justify-between">
  <a class="f4 fw4 hover-white white-70 dn dib-ns pv2 ph3 no-underline" href="https://lcornet.github.io/" >
    &copy;  Luc Cornet 2026 
  </a>
    <div><div class="ananke-socials"></div>
</div>
  </div>
</footer>

  </body>
</html>
